validator/src/main/kotlin/br/com/guiabolso/hyperloop/schemas/SchemaRepository.kt

```diff
@@ -2,8 +2,8 @@ package br.com.guiabolso.hyperloop.schemas
 
 import br.com.guiabolso.hyperloop.schemas.exceptions.SchemaFetchingException
 
-interface SchemaRepository {
+interface SchemaRepository<out R> {
```

Comment:
Pra quê serve o `out`?

Reply:
https://kotlinlang.org/docs/reference/generics.html#variance

Reply:
Got it! Valeu pelo link/explicação, amados!